vite.config.ts

```diff
@@ -19,20 +20,22 @@ export default defineConfig(async () => {
         }),
       },
       react(),
+      TanStackRouterVite(),
     ],
     test: {
       globals: true,
       environment: 'jsdom',
       setupFiles: './tests/setup.js',
       coverage: {
         exclude: [
-          '**/{*.d.ts,index.ts}',
+          '**/{*.d.ts,index.ts,*.gen.ts}',
```

Comment:
what files are they?

Reply:
Tanstack router create a routeTree.gen.ts. In this file are basically all routes listed which are generated from the routes directory. This file is automatically build when starting the dev server or build the production build, so we don't need to commit it.